libs/utils/utils.js

```diff
@@ -656,6 +656,23 @@ export function decorateLinks(el) {
         window.adobeIMS?.signIn();
       });
     }
+    const copyEvent = '#_evt-copy';
+    const isMobile = /android|iphone|ipod|blackberry|iemobile|opera mini|mobile/.test(navigator.userAgent.toLowerCase());
+    if (a.href.includes(copyEvent)) {
+      const link = a.href.split(copyEvent)[0];
+      const isConButton = ['EM', 'STRONG'].includes(a.parentElement.nodeName) || a.classList.contains('con-button');
+      if (!isConButton) a.classList.add('static', 'copy-link');
+      a.href = '';
+      if (isMobile) {
+        a.addEventListener('click', async (e) => {
+          e.preventDefault();
+          if (navigator.share) await navigator.share({ title: link, url: link });
+        });
+      } else {
+        a.remove();
+        console.log('Copy link is not supported for this device');
+      }
+    }
```

Comment:
I'd probably restructure this a little, untested demo code:
```suggestion
    const copyEvent = '#_evt-copy';
    const decorateCopyEvt = () => {
      const isMobile = /android|iphone|ipod|blackberry|iemobile|opera mini|mobile/.test(navigator.userAgent.toLowerCase());
      if (!isMobile) {
        a.remove();
        console.log('Copy link is not supported for this device');
        return;
      }
      const link = a.href.split(copyEvent)[0];
      const isConButton = ['EM', 'STRONG'].includes(a.parentElement.nodeName) || a.classList.contains('con-button');
      if (!isConButton) a.classList.add('static', 'copy-link');
      a.href = '';
      a.addEventListener('click', async (e) => {
        e.preventDefault();
        if (navigator.share) await navigator.share({ title: link, url: link });
      });
    };
```

Reply:
Sure, made these changes. Thanks!